blueprints/helper-addon/index.js

```diff
@@ -0,0 +1,3 @@
+/*jshint node:true*/
+
+module.exports = require('ember-cli/blueprints/addon-import');
```

Comment:
hmm, I'm not sure if that's so easy. `addon-import` is used by default in `ember-cli` so we probably shouldn't remove it there but I'm not a big fan of duplication either.

then again since it's the default there I'm not sure why this `helper-addon` blueprint is even needed if it just passes on to the default. the only advantage would be to be able to generate it directly with `ember g helper-addon`.